app/src/Containers/EventSearchContainer.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/native'
+import { ScrollView, SafeAreaView, Button, Text } from 'react-native'
+import TopOfApp from '@/Components/TopOfApp'
+import underDevelopmentAlert from '@/Utils/UnderDevelopmentAlert'
+import DateTimePicker from '@react-native-community/datetimepicker'
```

Comment:
In the team discussion we had the other week about the date picker, was the consensus that this would be most useful for the user to be able to pick a date range (e.g. find all the events in July) or was it to pick a single date?  I'd thought it was a date range, but that might just be my dodgy memory! 🙂

Reply:
No you're right but because so much had to be added to get to get somewhere to test date pickers, I wanted to get those commits upstream (if that makes sense).

Reply:
Replaced with a date range picker.